src/App.jsx

```diff
@@ -15,6 +15,7 @@ function App() {
 
   const [isFileMenuOpen, setIsFileMenuOpen] = useState(false);
   const [activeFileId, setActiveFileId] = useState(null)
+  const [tabName, setTabName] = useState(null)
```

Comment:
No need for the `useState` call, just derive it from what's already there.
```suggestion
  const tabName = (data && activeFileId) ? data[activeFileId].name : "";
```